src/test/java/nextstep/subway/unit/section/SectionListTest.java

```diff
@@ -143,9 +143,77 @@ void getUpLastStationWhenAddUpwardStation() {
         SectionList sectionList = line.getSections();
         Section newSection = new Section(line, new SectionStations(c, a), 4);
         sectionList.addSection(newSection);
+        Section newSection1 = new Section(line, new SectionStations(a, d), 1);
+        sectionList.addSection(newSection1);
 
-        Station lastUpstation = sectionList.getUpLastStation();
+        Station lastUpStation = sectionList.getUpLastStation();
+
+        assertThat(lastUpStation).isEqualTo(c);
+    }
+
+    @Test
+    void removeSection() {
+        SectionList sectionList = line.getSections();
+        Section newSection = new Section(line, new SectionStations(b, c), 4);
+        sectionList.addSection(newSection);
+
+        sectionList.removeSection(b);
+
+        assertThat(line.getDistance()).isEqualTo(9);
+        List<Section> sections = line.getSectionList();
+        assertThat(sections).hasSize(1);
+        Section leftSection = sections.get(0);
+        assertThat(leftSection.getDistance()).isEqualTo(9);
+        assertThat(leftSection.getUpwardStation()).isEqualTo(a);
+        assertThat(leftSection.getDownwardStation()).isEqualTo(c);
+    }
+
+    @Test
+    void removeSectionUpLastStation() {
+        SectionList sectionList = line.getSections();
+        Section newSection = new Section(line, new SectionStations(b, c), 4);
+        sectionList.addSection(newSection);
 
-        assertThat(lastUpstation).isEqualTo(c);
+        sectionList.removeSection(a);
+
+        assertThat(line.getDistance()).isEqualTo(4);
+        List<Section> sections = line.getSectionList();
+        assertThat(sections).hasSize(1);
+        Section leftSection = sections.get(0);
+        assertThat(leftSection.getDistance()).isEqualTo(4);
+        assertThat(leftSection.getUpwardStation()).isEqualTo(b);
+        assertThat(leftSection.getDownwardStation()).isEqualTo(c);
+    }
+
+    @Test
+    void removeSectionDownLastStation() {
+        SectionList sectionList = line.getSections();
+        Section newSection = new Section(line, new SectionStations(b, c), 4);
+        sectionList.addSection(newSection);
+
+        sectionList.removeSection(c);
+
+        assertThat(line.getDistance()).isEqualTo(5);
+        List<Section> sections = line.getSectionList();
+        assertThat(sections).hasSize(1);
+        Section leftSection = sections.get(0);
+        assertThat(leftSection.getDistance()).isEqualTo(5);
+        assertThat(leftSection.getUpwardStation()).isEqualTo(a);
+        assertThat(leftSection.getDownwardStation()).isEqualTo(b);
```

Comment:
단위 테스트에서는 객체끼리 비교해서 검증해 보시는 것도 고려해 보시면 좋을 것 같아요 😃 
```suggestion
        assertThat(leftSection).isEqualTo(new Section(line, new SectionStations(a, b), 5));
```

Reply:
음 궁금한게 저렇게 검증하려면 값에 관련된 Equal 메소드랑 HashCode를 새로 override 해야 하잖아요? 이 부분을 검증하기 위해서 추가해도 상관 없다는 것을 말씀하시는 것일까요?